packages/core/src/app/shipping/SingleShippingForm.test.tsx

```diff
@@ -229,31 +229,6 @@ describe('SingleShippingForm', () => {
         );
     });
 
-    it('calls updateAddress when address is updated but form is rendered before its initial value is loaded', async () => {
```

Comment:
This test is not valid any more as only two fields have values, it cannot pass validation to submit the form.